File: src/main-service/Doerly.Module.Profile/Doerly.Module.Profile.Contracts/Dtos/LanguageProficiencyDto.cs
using Doerly.Module.Profile.Enums;

namespace Doerly.Module.Profile.Contracts.Dtos;

public class LanguageProficiencyDto
{
    public int Id { get; set; }
    public required LanguageDto Language { get; set; }
    public required ELanguageProficiencyLevel Level { get; set; }
}

File: src/doerly-fe/src/app/modules/profile/models/responses/LanguageProficiencyDto.ts
import { LanguageDto } from './LanguageDto';
import { ELanguageProficiencyLevel } from '../enums/ELanguageProficiencyLevel';

export interface LanguageProficiencyDto {
  id: number;
  language: LanguageDto;
  level: ELanguageProficiencyLevel;
} 